kmip/pie/objects.py

```diff
@@ -227,6 +235,7 @@ def __init__(self, algorithm, length, value, masks=None,
         self.value = value
```

Comment:
Do we need a check to make sure values is actually a bytearray? I wish Python had type safety, so I do not know what is proper procedure.